tasks/elasticsearch-templates.yml

```diff
@@ -29,5 +29,7 @@
   register: resultstemplate
 
 - name: Install template(s)
-  command: "curl -sL -XPUT http://localhost:{{http_port}}/_template/{{item}} -d @/etc/elasticsearch/templates/{{item}}.json"
+  command: "curl -sL -XPUT --output /dev/null --write-out %{http_code} http://localhost:{{http_port}}/_template/{{item}} -d @/etc/elasticsearch/templates/{{item}}.json"
```

Comment:
@aosagie thanks for pull request. I'm doing a larger pull which moves templates to a handler. I'll use the uri module as suggested by @barryib. Thanks for the reminder.